src/Integration.Vsix.UnitTests/InfoBar/InfoBarManagerTests.cs

```diff
@@ -303,6 +303,29 @@ public void AttachInfoBarWithButtons_ButtonClickedEvent_RaisedWithTheRightButton
             eventHandler.VerifyNoOtherCalls();
```

Comment:
Also while looking at this class, I found there are quite a few Configurable... test helper classes that we do not use. Maybe it would make sense to delete them simply to avoid anyone reusing them in the future. (For example the class  ConfigurableInfoBar.cs )

Reply:
Update: I've deleted two of the unused `ConfigurableXXX` classes